src/LinuxTracing/ThreadStateManagerTest.cpp

```diff
@@ -275,6 +314,8 @@ TEST(ThreadStateManager, UnknownInitialStateWithSwitchIn) {
 
   slice = manager.OnSchedSwitchIn(100, kTid);
   EXPECT_FALSE(slice.has_value());
+  EXPECT_EQ(slice->woker_pid(), 0);
+  EXPECT_EQ(slice->woker_tid(), 0);
```

Comment:
The `slice` optional is empty. We shouldn't access the members here. Should this rather go down to the other checks?